STAT/TStatToolkit.cxx

```diff
@@ -1753,6 +1755,10 @@ TMultiGraph*  TStatToolkit::MakeStatusMultGr(TTree * tree, const char * expr, co
     snprintf(query,200, "%f*(%s-0.5):%s", 1.+igr, oaAlias->At(i)->GetName(), var_x);
     TGraphErrors * gr = (TGraphErrors*) TStatToolkit::MakeGraphSparse(tree,query,cut,marArr[i],colArr[i],sizeArr[i],shiftArr[i]);
     if (gr) multGr->Add(gr);
+    else{
+        ::Error("MakeGraphSparse"," returned with error -> returning");
+        return 0;
```

Comment:
adjust logic according error handling parameter.
accept or  do  not accept partial multi-graphs 
Options:
* all graphs requered
* at minimum one graph required
Should return 0 pointer if not fulfilled condition